data/net/minecraft/commands/arguments/item/FunctionArgument.mapping

```diff
@@ -1,4 +1,24 @@
 CLASS net/minecraft/commands/arguments/item/FunctionArgument
+	METHOD getFunction (Lcom/mojang/brigadier/context/CommandContext;Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/commands/CommandFunction;
+		ARG 0 context
+		ARG 1 resourceLocation
+	METHOD getFunctionOrTag (Lcom/mojang/brigadier/context/CommandContext;Ljava/lang/String;)Lcom/mojang/datafixers/util/Pair;
+		ARG 0 context
+		ARG 1 name
+	METHOD getFunctionTag (Lcom/mojang/brigadier/context/CommandContext;Lnet/minecraft/resources/ResourceLocation;)Lnet/minecraft/tags/Tag;
+		ARG 0 context
+		ARG 1 resourceLocation
```

Comment:
A better name for the `ResourceLocation` parameters would be `id` or `functionId`, instead of the generic `resourceLocation`.